pkg/util/node/node.go

```diff
@@ -151,9 +151,36 @@ func SetNodeCondition(c clientset.Interface, node types.NodeName, condition v1.N
 
 // PatchNodeStatus patches node status.
 func PatchNodeStatus(c clientset.Interface, nodeName types.NodeName, oldNode *v1.Node, newNode *v1.Node) (*v1.Node, error) {
+	patchBytes, err := preparePatchBytesforNodeStatus(nodeName, oldNode, newNode)
+	if err != nil {
+		return nil, err
+	}
+
+	updatedNode, err := c.Core().Nodes().Patch(string(nodeName), types.StrategicMergePatchType, patchBytes, "status")
+	if err != nil {
+		return nil, fmt.Errorf("failed to patch status %q for node %q: %v", patchBytes, nodeName, err)
+	}
+	return updatedNode, nil
+}
+
+// UpdateNodeStatus updates node status.
+func UpdateNodeStatus(c clientset.Interface, nodeName types.NodeName, oldNode *v1.Node, newNode *v1.Node) (*v1.Node, error) {
+	patchBytes, err := preparePatchBytesforNodeStatus(nodeName, oldNode, newNode)
+	if err != nil {
+		return nil, err
+	}
+
+	updatedNode, err := c.CoreV1().Nodes().PatchStatus(string(nodeName), patchBytes)
```

Comment:
Why you use CoreV1 here?

Actually I feel these two functions PatchNodeStatus and UpdateNodeStatus are the same. We only need one of those.

Reply:
Will remove one.

> Why you use CoreV1 here?

`Core` is deprecated. Do we still use it instead of `CoreV1`?

Reply:
I am not sure about this actually. Maybe we should confirm with someone in api team.

cc @caesarxuchao Should we use CoreV1 or just Core?

Reply:
CoreV1 is better.

Reply:
thanks!
@dixudx could you use CoreV1 instead based on @caesarxuchao 's suggestion?

Reply:
@jingxu97 @caesarxuchao Done. PTAL. Thanks.